docs/developers_guide/cnvmap_struct_c.md

```diff
@@ -0,0 +1,94 @@
+<!--
+(C) copyright 2020 VT SuperDARN, Virginia Polytechnic Institute & State University
+author: Kevin Sterne
+-->
+
+# Structure in C for the Convection Map data
+
+## Summary
+
+A description of the data structure in the C language used to store the convection map data.
+
+## Description
+
+### Introduction
+
+SuperDARN Convection map data consists of a superset of the gridded data. Listed below is the C data structure <code>struct CnvMapData</code> used to store the gridded data.
+
+### `struct CnvMapData`
```

Comment:
In readthedocs it looks less like a subsection, are you okay with how it looks?

Doesn't bother me but didn't think there was another section to Introduction

Reply:
I was struggling with how to best represent those struct headings.  I think it looks OK, but maybe some others could weigh in?  I'm open to another way those could be represented.